muckrock/agency/api_v2/serializers.py

```diff
@@ -69,3 +69,35 @@ class Meta:
             "parent",
             "appeal_agency",
         )
+        extra_kwargs = {
+            "id": {"help_text": "The unique identifier for this agency."},
+            "name": {"help_text": "The name of the agency."},
+            "slug": {"help_text": "The slug (URL identifier) for the agency."},
+            "status": {
+                "help_text": (
+                    "The current status of the agency (e.g., approved, pending, rejected)."
+                )
+            },
+            "exempt": {
+                "help_text": (
+                    "Indicates whether the agency is exempt from records laws "
+                    "(True/False)."
+                )
+            },
+            "types": {
```

Comment:
The documentation doesn't seem to be picking up the help text for types for some reason